test/bin/mdx-pp/expect/spaces/test_case.ml.expected

```diff
@@ -1,4 +1,4 @@
-#5 "test-case.md"
+#11 "test-case.md"
```

Comment:
Is this expected?

Reply:
No it's not, but it looks like the whole situation is a mess, the locations are guessed and modified everywhere!

Reply:
fixed (but the fix is ugly)